### ISSUE TITLE
distroless/java:11 missing liblcms2.so.2

### ISSUE THREAD

**Issue by mgubaidullin** [GoogleContainerTools/distroless#329]
**Description of the issue**: Missing liblcms2.so.2 error when use apache-poi (apache-poi uses java.awt.color.*). The docker images build ok, but I get an exception thrown when the container runs. The application runs ok in the IDE against java 11.

**Expected behavior**: No exception

**Steps to reproduce**: I have created a tiny example application to reproduce this issue https://github.com/mgubaidullin/example

```
git clone https://github.com/mgubaidullin/example.git
cd example
mvn clean compile jib:dockerBuild
docker run -i --rm -p 8080:8080 gcr.io/example/lcms
```

**Environment**: Linux / Java 11 

**Log output**:
```
Exception in thread "main" java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/libjavalcms.so: liblcms2.so.2: cannot open shared object file: No such file or directory
        at java.lang.ClassLoader$NativeLibrary.load(Native Method)
        at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941)
        at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1845)
        at java.lang.Runtime.loadLibrary0(Runtime.java:870)
        at java.lang.System.loadLibrary(System.java:1122)
        at sun.java2d.cmm.lcms.LCMS$1.run(LCMS.java:210)
        at java.security.AccessController.doPrivileged(Native Method)
        at sun.java2d.cmm.lcms.LCMS.getModule(LCMS.java:203)
        at sun.java2d.cmm.lcms.LcmsServiceProvider.getModule(LcmsServiceProvider.java:34)
        at sun.java2d.cmm.CMMServiceProvider.getColorManagementModule(CMMServiceProvider.java:31)
        at sun.java2d.cmm.CMSManager.getModule(CMSManager.java:76)
        at java.awt.color.ICC_ColorSpace.toRGB(ICC_ColorSpace.java:177)
        at example.LittleCms.main(LittleCms.java:9)

```

**How to fix**: I have fixed this issue locally by cloning distroless repository and adding 
`packages["liblcms2-2"],` into BUILD file and
`"liblcms2-2",` into WORKSPACE

I could make a PR if needed

**Comment by chanseokoh** [comment]
Awesome! Looks like you already know the stuff. We'd appreciate your PR.

**Comment by chanseokoh** [comment]
Fixed by #333. Closing.
